AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_DepthOfFieldAdded.py

```diff
@@ -175,29 +326,404 @@ def AtomEditorComponents_DepthOfField_AddedToEntity():
+                AtomComponentProperties.depth_of_field('Quality Level')) == 1)
+
+        # 27. Aperture F can be set to 0 and back to 0.5.
+        depth_of_field_component.set_component_property_value(
+            AtomComponentProperties.depth_of_field('Aperture F'), 0)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.aperturef_override_set_to_0,
+            depth_of_field_component.get_component_property_value(
+                AtomComponentProperties.depth_of_field('Aperture F')) == 0)
+
+        depth_of_field_component.set_component_property_value(
+            AtomComponentProperties.depth_of_field('Aperture F'), .5)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.aperture_f_set_to_five_tenths,
+            depth_of_field_component.get_component_property_value(
+                AtomComponentProperties.depth_of_field('Aperture F')) == .5)
+
+        # F Number Note - Discussed it with the dev owner, and we determined this does not need to be tested.
+
+        # 28a. Enable Auto Focus can be turned off.
+        # Turn off Auto Focus to unlock Focus distance setting
+        depth_of_field_component.set_component_property_value(
+            AtomComponentProperties.depth_of_field('Enable Auto Focus'), False)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.enable_auto_focus_off,
+            depth_of_field_component.get_component_property_value(
+                AtomComponentProperties.depth_of_field('Enable Auto Focus')) is False)
+
+        # 29. Focus Distance can be set to 0 then back to 100.
+        depth_of_field_component.set_component_property_value(
+            AtomComponentProperties.depth_of_field('Focus Distance'), 0)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.focus_distance_set_to_0,
+            depth_of_field_component.get_component_property_value(
+                AtomComponentProperties.depth_of_field('Focus Distance')) == 0)
+
+        depth_of_field_component.set_component_property_value(
+            AtomComponentProperties.depth_of_field('Focus Distance'), 100)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.focus_distance_set_to_100,
+            depth_of_field_component.get_component_property_value(
+                AtomComponentProperties.depth_of_field('Focus Distance')) == 100)
+
+        # 28b. Enable Auto Focus can be turned back on.
+        depth_of_field_component.set_component_property_value(
+            AtomComponentProperties.depth_of_field('Enable Auto Focus'), True)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.enable_auto_focus_on,
+            depth_of_field_component.get_component_property_value(
+                AtomComponentProperties.depth_of_field('Enable Auto Focus')) is True)
+
+        # 30. Focus Screen Position.
+        position_vector_0 = math.Vector2(0.0, 0.0)
```

Comment:
just curious but is this top left or bottom right?

Reply:
top left